.github/workflows/pr-windows-ae.yml

```
@@ -119,17 +121,25 @@ jobs:
       with:
         name: dist
         path: ${{github.workspace}}/build/dist
-
     - uses: actions/upload-artifact@v3
       with:
         name: coverage
         path: ${{github.workspace}}/build/__coverage
 
+      # - run: |
+      #     New-Item -ItemType Directory -Path ${{github.workspace}}/build/nexus/sp/data/ -Force
+      #     New-Item ${{github.workspace}}/build/nexus/sp/data/foo.bar.txt
     - uses: actions/upload-artifact@v3
       with:
         name: SP-AE
         # Data folder is skipped for mod managers
         path: ${{github.workspace}}/build/nexus/sp/data/*
+    # create a zip archive
+    - run: |
+        Compress-Archive -Path ${{github.workspace}}/build/nexus/sp/data -DestinationPath ${{github.workspace}}/build/nexus/sp.zip
+    - run: |
+        dotnet tool install -g Digitalroot.ModUploader
+        drmu nexusmods upload 54909 ${{github.workspace}}/build/nexus/sp/data --file-name "Skyrim Platform 2.7.0 (Anniversary Edition)" --version "2.7.0" --category Main --description "THIS IS FOR ANNIVERSARY EDITION (1.6.x)!\n\nPlease backup your saves before updating. There are issues with some modlists.\n\nChangelog: https://github.com/skyrim-multiplayer/skymp/blob/main/docs/release/sp-2.7.0.md\n\nFor questions: https://discord.gg/k39uQ9Yudt" --game skyrimspecialedition
```

I think the script is failing because of this:
```suggestion
        drmu nexusmods upload 54909 ${{github.workspace}}/build/nexus/sp.zip --file-name "Skyrim Platform 2.7.0 (Anniversary Edition)" --version "2.7.0" --category Main --description "THIS IS FOR ANNIVERSARY EDITION (1.6.x)!\n\nPlease backup your saves before updating. There are issues with some modlists.\n\nChangelog: https://github.com/skyrim-multiplayer/skymp/blob/main/docs/release/sp-2.7.0.md\n\nFor questions: https://discord.gg/k39uQ9Yudt" --game skyrimspecialedition
```
Correct me if I'm wrong 💯

---

Looks like it, yeah